sktime/annotation/base/_mock.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+from sktime.annotation.base._base import BaseSeriesAnnotator
+
+
+class MockSeriesAnnotator(BaseSeriesAnnotator):
```

Comment:
Since we have the `PyODAnnotator` in this PR, I think we can remove the mock class

Reply:
will do once the `PyODAnnotator` works